content/quickstart.md

```diff
@@ -48,43 +123,31 @@ $ minishift config set memory 8384
 
-* **Tester** / **User** : In case you only want to have the latest version of Syndesis on your local Minishift installation, use the template [`syndesis-restricted`](https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/syndesis-restricted.yml) which uses image stream referring to the published Docker Hub images. Minishift will update its images and trigger a redeployment when the images at Docker Hub changes. Therefore it checks every 15 minutes for a changed image. You do not have to do anything to get your application updated, except for waiting on Minishift to pick up new images.
+* **Developer** : Use the template [`syndesis-dev`](https://raw.githubusercontent.com/syndesisio/syndesis/master/install/syndesis-dev.yml) which directly references Docker images without image streams. Then before building the images e.g. with `mvn fabric8:build` set your `DOCKER_HOST` environment variable to use the Minishift Docker daemon via `eval $(minishift docker-env)`. When new images are built you only need to delete the appropriate pod so that the new pod spinning up will use the freshly built image.
 
 Depending on your role please use the appropriate template in the instructions below.
 
-### Deployment instructions
-
-Before we can start installing Syndesis we have to add some policies
-
-```bash
-$ oc login -u system:admin
-$ oc adm policy add-cluster-role-to-user system:auth-delegator -z syndesis-oauth-client
-$ oc login -u developer
-```
-
-Install the OpenShift template (syndesis-dev-restricted.yml or syndesis-restricted.yml as discussed [above]({{< relref "#template-selection" >}})):
+Install the OpenShift template `syndesis-dev.yml` or `syndesis.yml` as discussed [above]({{< relref "#template-selection" >}})):
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/syndesis-dev-restricted.yml
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/install/syndesis.yml
 ```
 
 In order to make it easy to run Syndesis on a cluster without requiring admin rights, Syndesis takes advantage of OpenShift's ability to use a [Service Account as an OAuth client](https://docs.openshift.org/latest/architecture/additional_concepts/authentication.html#service-accounts-as-oauth-clients). Before we create the app, we'll need to create this Service Account:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/app/deploy/support/serviceaccount-as-oauthclient-restricted.yml
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/install/support/serviceaccount-as-oauthclient-restricted.yml
 ```
 
 Deploy syndesis using the following command, replacing "syndesis-dev-restricted" with "syndesis-restricted" depending on the template
```

Comment:
Let's capitalize Syndesis here

Reply:
good point, also `-restricted` is not anymore.